Content.Server/Defusable/Systems/DefusableSystem.cs

```diff
@@ -1,22 +1,81 @@
 using Content.Server.Defusable.Components;
 using Content.Server.Explosion.EntitySystems;
+using Content.Server.Popups;
 using Content.Server.Salvage;
 using Content.Shared.Defusable;
 using Content.Shared.Examine;
+using Content.Shared.Verbs;
+using Robust.Shared.Audio;
+using Robust.Shared.Player;
 
 namespace Content.Server.Defusable.Systems;
 
 /// <inheritdoc/>
 public sealed class DefusableSystem : SharedDefusableSystem
 {
     [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
-    [Dependency] private readonly ExplosionSystem _explosionSystem = default!;
+    [Dependency] private readonly ExplosionSystem _explosion = default!;
+    [Dependency] private readonly PopupSystem _popup = default!;
 
     /// <inheritdoc/>
     public override void Initialize()
     {
         base.Initialize();
         SubscribeLocalEvent<DefusableComponent, ExaminedEvent>(OnExamine);
+        SubscribeLocalEvent<DefusableComponent, GetVerbsEvent<AlternativeVerb>>(OnGetAltVerbs);
+    }
+
+    /// <summary>
+    ///     Add an alt-click interaction that cycles through delays.
+    /// </summary>
+    private void OnGetAltVerbs(EntityUid uid, DefusableComponent comp, GetVerbsEvent<AlternativeVerb> args)
+    {
+        if (!args.CanInteract || !args.CanAccess)
+            return;
+
+        args.Verbs.Add(new AlternativeVerb()
+        {
+            Text = Loc.GetString("defusable-verb-begin"),
+            Disabled = false,
+            Priority = 10,
+            Act = () =>
+            {
+                TryStartCountdown(uid, comp);
+            }
+        });
+
+        if (comp.StartingTimeOptions == null || comp.StartingTimeOptions.Count == 1)
+            return;
+
+
+        foreach (var option in comp.StartingTimeOptions)
```

Comment:
why are you reimplementing timer trigger

Reply:
i don't know why either

Reply:
timter trigger